mods/netstimbox.mod

```diff
@@ -5,7 +5,7 @@
 
 NEURON	{ 
   ARTIFICIAL_CELL NetStimBox
-  RANGE start, forcestop, status, nspk
+  RANGE start, forcestop, status, nspk, freq
```

Comment:
is this change doing anything in this file, or are you relying on another .mod now?